ToDo/src/main/java/com/todo/demo/Service/NoteServiceHandler.java

```diff
@@ -0,0 +1,59 @@
+package com.todo.demo.Service;
+
+import com.todo.demo.Note.Note;
+import com.todo.demo.thrift.gen.NoteServiceThrift;
+import com.todo.demo.thrift.gen.NoteThrift;
+import org.apache.thrift.TException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class NoteServiceHandler implements NoteServiceThrift.Iface {
+
+    @Autowired
+    private NoteService noteService;
+
+    public NoteServiceHandler(NoteService noteService) {
+        this.noteService = noteService;
+    }
+
+    @Override
+    public String sayHello() throws TException {
+        return noteService.sayHello();
+    }
+
+    @Override
+    public int createNote(NoteThrift note) throws TException {
+        Note n = new Note();
+        n.toNote(note);
+        noteService.createNote(n);
+        return n.getId();
+    }
+
+    @Override
+    public List<NoteThrift> readNote() throws TException {
+        List<NoteThrift> noteThriftList = new ArrayList<>();
+
+        for (Note n:noteService.readNote()) {
```

Comment:
instead could be done as `return noteService.read().stream().map(Note::toThriftNote).collect(Collectors.toList())`

Reply:
Any specific advantage of using this?